docs/developers/deploy-on-polygon.md

```diff
@@ -0,0 +1,143 @@
+
+
+```bash
+wss://polygon-mumbai.infura.io/v3/{projectId}
+```
+
+If using Infura for a mainnet fork, `YOUR_NODE_URL` will follow this format:
+
+```bash
+https://polygon-mumbai.infura.io/v3/{projectId}
+```
+
+Run the deployment script with your specific parameters.
+```bash
+node index.js --gasprice 80 --url YOUR_NODE_URL --mnemonic "your mnemonic (12 word seed phrase)" --lspCreatorAddress 0x566f98ECadE3EF95a6c5840621C43F15f403274c --pairName "UMA \$4-12 Range Token Pair August 2021" --expirationTimestamp 1630447200 --collateralPerPair 250000000000000000 --priceIdentifier UMAUSD --longSynthName "UMA \$4-12 Range Token August 2021" --longSynthSymbol rtUMA-0821 --shortSynthName "UMA \$4-12 Range Short Token August 2021" --shortSynthSymbol rtUMA-0821s --collateralToken 0x489Bf230d4Ab5c2083556E394a28276C22c3B580 --customAncillaryData "twapLength:3600" --optimisticOracleLivenessTime 3600 --fpl RangeBond
+```
+## Deploying to Polygon Mainnet
+
+The method to deploy onto mainnet is the exact same as with Mumbai, with a few key items to note. Your `url`, `lspCreatorAddress` and `financialProductLibrary` will need to be amended for use on Polygon mainnet. Since the library addresses are different between the networks. 
+
+All the required Polygon Mainnet addresses can be seen on our [Github repo](https://github.com/UMAprotocol/protocol/blob/master/packages/core/networks/137.json) with the most important items in the list for specific launch are:
+
+- `Polygon LongShortPairCreator` 
+- `BinaryOptionLongShortPairFinancialProductLibrary` 
+- `CoveredCallLongShortPairFinancialProductLibrary` 
+- `LinearLongShortPairFinancialProductLibrary` 
+- `RangeBondLongShortPairFinancialProductLibrary` 
+
+Please note that the script will deploy to the network that your node URL is for. If using Infura for a Polygon deployment, `YOUR_NODE_URL` will follow this format:
+
+
+```bash
+wss://polygon-mainnet.infura.io/v3/{projectId}
+```
+
+If using Infura for a mainnet fork, `YOUR_NODE_URL` will follow this format:
+
+```bash
+https://polygon-mainnet.infura.io/v3/{projectId}
+```
+
+You can now run the deployment script. From within the `launch-lsp` directory, run:
+```bash
+node index.js --gasprice 80 --url YOUR_NODE_URL --mnemonic "your mnemonic (12 word seed phrase)" --lspCreatorAddress 0x566f98ECadE3EF95a6c5840621C43F15f403274c --pairName "UMA \$4-12 Range Token Pair August 2021" --expirationTimestamp 1630447200 --collateralPerPair 250000000000000000 --priceIdentifier UMAUSD --longSynthName "UMA \$4-12 Range Token August 2021" --longSynthSymbol rtUMA-0821 --shortSynthName "UMA \$4-12 Range Short Token August 2021" --shortSynthSymbol rtUMA-0821s --collateralToken 0x489Bf230d4Ab5c2083556E394a28276C22c3B580 --customAncillaryData "twapLength:3600" --optimisticOracleLivenessTime 3600 --fpl RangeBond
```

Comment:
Sorry, I know we want to get this merged. Some of these parameters are specific to Kovan. I know this is how they are included in the launch-lsp repo, but it seems confusing if we are providing an example command with kovan contract addresses for a polygon/mumbai tutorial.

Reply:
cool, makes sense. added